rpcs3/Emu/RSX/GSRender.h

```diff
@@ -69,6 +69,11 @@ class GSFrameBase
 	virtual void hide() = 0;
 	virtual void show() = 0;
 
+	// taskbar progress
+	virtual void progress_reset() = 0;
```

Comment:
This is unnecessary. These methods belong to the GSFrame implementation, only the core stuff should be in the frame base class (i.e, it is not necessarily true that the implementation has support for taskbar progress at all). It will work just fine once you remove the override specifier in the implementation methods.